### DISCUSSION TITLE
Feature request: Change Font atributes

### DISCUSSION THREAD

**Discussion by CommanderB** [ybizeul/StreamDeckWS#18] [Ideas]
Would it be possible to make the font, size, and colour of the title text changeable? 

In addition to the ability to change the title text, changing the colour and size would be useful mechanisms for feedback on the Stream Deck. 

Thanks for listening!

**Comment by ybizeul** [comment]
Ha, now that wouldn't be possible, it's not part of Stream Deck APIs ! You have to change color and size in the Stream Deck app and there is no way that I know of to change that remotely.

**Reply by CommanderB** [reply]
Got it. Thanks
